### ISSUE TITLE
Request for Ubuntu 18.04 package

### ISSUE THREAD

**Issue by dboune** [holland-backup/holland#196]
Hi! Requesting package build for Ubuntu 18.04 please! Thank you!

**Comment by soulen3** [comment]
Yes, I plan on trying to get something released for Ubuntu 18.04.  But I don't know much about Debian packaging, and I'll be working on the EL6/7 RPMs first.

For now you could try out the packages built here:
https://build.opensuse.org/project/repository_state/home:holland-backup/xUbuntu_18.04
